File: fibonacci.c
#include<stdio.h>
int fibonacci(int x);
int main() {
    int num=0;
    printf("enter the number: ");
    scanf("%d", &num);
    fibonacci(num);
}
int fibonacci(int x) {
    int a=0, b=1, c;
    printf("%d ", a);
    printf("%d ", b);
    for(int i=1; i<=x; i++) {
        c=a+b;
        printf("%d ", c);
        a=b;
        b=c;
    }
}

File: fibonacci.java
public class fibonacci {
    static int n1 = 0, n2 = 1, n3 = 0;

    public static void fibo(int count) {

        if (count > 0) {
            n3 = n1 + n2;
            n1 = n2;
            n2 = n3;
            System.out.print(n3 + " ");
            fibo(count - 1);
        }
    }

    public static void main(String args[]) {
        // System.out.print(n1 + " " + n2 + " ");
        fibo(10);
    }
}
